setup.py

```diff
@@ -30,10 +30,6 @@ def find_version(*file_paths):
         'requests>=2.7,<3.0',
     ],
 
-    extras_require = {
```

Comment:
this would rely on client application defining all dependencies, is that the desired behavior? is this mostly because we are relying on the client application to provide the `auth=HTTPKerberosAuth()`?

Reply:
Correct, basically whole kerberos change is just 2 lines in codebase. We can completely eliminate it by asking user to install it. Another possible aapproach is but keeping extra package, but have no code that actually uses imports from it, since it user actually injects auth